docs/proposal/ipsec_e2e_zh..md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
FYI:

Design doc of Kmesh E2E framework: https://github.com/kmesh-net/kmesh/blob/main/docs/proposal/e2e.md

User guide: https://kmesh.net/en/docs/developer/e2e-guide/

and all test cases should be added in https://github.com/kmesh-net/kmesh/tree/main/test/e2e

You can write new test cases based on existing ones.

Reply:
Also English doc is prefered